pkg/config/nodetreemodel/config_test.go

```diff
@@ -386,6 +386,19 @@ func TestEnvVarMultipleSettings(t *testing.T) {
 	assert.Equal(t, 0, cfg.GetInt("c"))
 }
 
+func TestEmptyEnvVarSettings(t *testing.T) {
+	cfg := NewNodeTreeConfig("test", "TEST", nil)
+	cfg.SetDefault("a", -1)
+	cfg.BindEnv("a")
+	t.Setenv("TEST_A", "")
```

Comment:
I think it's worth adding a comment here explaining exactly what this test is demonstrating. The semantics are not completely obvious otherwise.
```suggestion
	// This empty string is ignored, so the default value of -1 will be returned by GetInt
	t.Setenv("TEST_A", "")
```